core/store/src/trie/ops/interface.rs

```diff
@@ -136,6 +142,12 @@ pub struct GenericTrieNodeWithSize<GenericTrieNodePtr, GenericValueHandle> {
     pub memory_usage: u64,
 }
 
+impl<N, V> Default for GenericTrieNodeWithSize<N, V> {
```

Comment:
Default value is empty node, needed if our retrieve node returns None. Otherwise too, good to have this implemented. `#derive` doesn't work here due to generics.